src/TutorialTemplate.js

```diff
@@ -1,48 +1,72 @@
-import React from 'react';
-import styled from 'styled-components/macro';
-import meditation from './assets/image-meditation.jpg'
+import React from 'react'
+import styled from 'styled-components/macro'
+import data from './data/content.json'
 
-
-export default function TutorialTemplate (){
-    return (
-    <TutorialWrapper>
-        <img src={meditation} alt=""/>
-        <h2>5 Minuten of...</h2>
-        <p>Lorem ipsum dolor sit amet consectetur Quos accusamus libero minus ullam, beataelibero minus ullam, beatae possimus nostr</p>
-    </TutorialWrapper>
-    )   
+export default function TutorialTemplate() {
+  return (
+    <>
+      {data.map(
+        ({
+          headline,
+          title,
+          image,
+          description1,
+          description2,
+          description3,
+          id,
+        }) => (
+          <TutorialWrapper key={id}>
+            <img src={image} alt="" />
+            <h2>{headline}</h2>
+            <h3>{title}</h3>
+            <p>{description1}</p>
+            <p>{description2}</p>
+            <p>{description3}</p>
+          </TutorialWrapper>
+        )
+      )}
+    </>
+  )
 }
 
 const TutorialWrapper = styled.div`
-    display: grid;
-    border: 1px solid var(--primary-border);
-    background-color: var(--secendary-background); 
-    box-shadow:  20px 20px 60px var(--primary-shadow),
-                -10px -10px 50px var(--secendary-shadow);
-    margin-left: 60px;
-    margin-top: 30px;
-    padding:0 15px 20px 15px;
-    place-items: center;
-    width: 70%;
+  display: grid;
+  border: 1px solid var(--primary-border);
+  background-color: var(--secondary-background);
+  box-shadow: 20px 20px 60px var(--primary-shadow),
+    -10px -10px 50px var(--secondary-shadow);
+  margin-left: 60px;
```

Comment:
`margin-left` looks a bit weird, the enclosing container (in `App.js` in this case) could just manage the position of this element

Reply:
done